test/features/steps/step_bootstrap.py

```diff
@@ -0,0 +1,108 @@
+import re
+import logging
+from behave import given, when, then
+from crmsh import utils, bootstrap, corosync, parallax
+
+
+def run_command(context, cmd):
```

Comment:
Why now it's not maybe needed, I would prefer to only have `steps` code in `steps` files. I would put this auxiliar methods in some other file.

Reply:
changed
I put them into utils.py file